src/components/Jobs/Form/Form.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types'
```

Comment:
Please move this Form component into the forms directory with other forms and give the Form a meaningful name vs Form.js

Reply:
Done